### PR TITLE
Synchronization improvements

### PR DESCRIPTION
**Pull request by ajardin**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -78,6 +78,10 @@ start: ## Start the environment
 		--sync-mode="two-way-resolved" \
 		--ignore-vcs --ignore=".idea" \
 		"${PROJECT_LOCATION}" "docker://${COMPOSE_PROJECT_NAME}_synchro/var/www/html/"
+	while [[ ! $$(mutagen list) =~ 'Status: Watching for changes' ]]; do \
+		echo 'Waiting for synchronization to complete...'; \
+		sleep 10; \
```
Comment (Fulmenef) [inline Makefile]: Isn't 10 seconds a bit long ?

Comment (ajardin) [reply Makefile]: The initial synchronization takes between 2 and 3 minutes to complete. It will increase the spam if I reduce the sleep duration. 😄

Comment (ajardin) [reply Makefile]: And the restart process takes between 40 and 50 seconds.

Comment (Fulmenef) [reply Makefile]: Okay, good enough for me :)

**Pull request merged**